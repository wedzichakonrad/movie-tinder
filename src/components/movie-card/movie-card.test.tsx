import MovieCard from "./movie-card";
import renderer from 'react-test-renderer';

it('should match snapshot for movie card component', () => {
    const movie = {
        id: 'id1',
        imageURL: 'url',
        title: 'title',
        summary: 'lorem ipsum',
        rating: 4,
    };

    const component = renderer.create(
        <MovieCard movie={movie} />
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});