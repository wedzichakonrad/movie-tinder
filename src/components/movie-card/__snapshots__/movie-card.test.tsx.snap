// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot for movie card component 1`] = `
<div
  className="movie-card"
  draggable={true}
  onDragStart={[Function]}
>
  <div
    className="movie-card__inner"
  >
    <div
      className="movie-card__heading-wrapper"
    >
      <h2>
        title
      </h2>
      <span>
        (
        4
        /10)
      </span>
    </div>
    <div
      className="movie-card__img-wrapper"
    >
      <img
        alt="movie photo"
        src="url"
      />
    </div>
    <div
      className="movie-card__buttons-wrapper"
    >
      <button
        className="movie-card-button movie-card-button--accept"
        onClick={[Function]}
      >
        <div
          className="movie-card-button__icon-wrapper"
        >
          <svg>
            check.svg
          </svg>
        </div>
        <span>
          Accept
        </span>
      </button>
      <button
        className="movie-card-button movie-card-button--reject"
        onClick={[Function]}
      >
        <div
          className="movie-card-button__icon-wrapper"
        >
          <svg>
            cross.svg
          </svg>
        </div>
        <span>
          Reject
        </span>
      </button>
    </div>
  </div>
</div>
`;
